README.md

```diff
@@ -1 +1,4 @@
-# Hello-Word
\ No newline at end of file
+# Hello-Word
+In this project we wanna just great personne who will see our page (note : It's exercise)
+We call this project Greating
+We built this project just by using html and css
```

Comment:
- [ ] Great job setting up your first project at Microverse! 💯 However, please use [Microverse's README template](https://github.com/microverseinc/readme-template) in order to have professional documentation. Don't forget to customize it with your own project details! 🤓 🛠️